app/_landing_pages/terraform.yaml

```
@@ -1,18 +1,218 @@
   content_type: landing_page
-  description: This page is an introduction to the Terraform providers for {{site.base_gateway}} and {{site.konnect_short_name}}.
+  description: This page is an introduction to the Terraform providers for {{site.konnect_short_name}}, {{site.base_gateway}} and {{ site.mesh_product_name }}.
 
 rows:
   - header:
       type: h1
-      text: "Terraform providers for Kong"
+      text: "Kong ❤️ Terraform"
+
+  - columns:
+    - blocks:
+        - type: structured_text
+          config:
+            header:
+              text: "What is Terraform?"
+            blocks:
+              - type: text
+                text: |
+                  Terraform is an infrastructure as code tool from HashiCorp. Primarily used for managing
+                  infrastructure such as AWS, GCP or Azure deployments, Terraform can also be used to manage
+                  your Kong configuration.
+        - type: button
+          config:
+            text: Get Started with Terraform and Konnect &raquo;
+            url: "#TODO"
+    - blocks:
+        - type: structured_text
+          config:
+            header:
+              text: "When should I use Terraform?"
+            blocks:
+              - type: text
+                text: |
+                  A common question we hear from you all is:
+
+                  _"Why would I choose Terraform, and when would I choose [deck](/deck/)?"_
+
+                  You should choose Terraform if one or more of the following is true:
+
+                  * You have existing review and approval processes built around Terraform
+                  * Your internal users are already familiar with HCL / Terraform
+
 
   - header:
       type: h2
-      text: What is Terraform?
+      text: "Available Providers"
     columns:
-    - blocks:
+      - blocks:
+        - type: card
+          config:
+            icon: /assets/icons/mesh.svg
+            title: Konnect
+            description: |
+              This is the official Konnect Terraform provider. You can use it to configure Dedicated Cloud Gateways, Control Planes, Gateway Entities, Portal, Teams and more
+            featured: true
+            ctas:
+              - text: GitHub
+                url: "https://github.com/Kong/terraform-provider-konnect"
+              - text: Terraform Registry
+                url: "https://registry.terraform.io/providers/Kong/konnect/latest"
+              - text: Examples
+                url: "https://github.com/Kong/terraform-provider-konnect/tree/main/examples/scenarios"
+      - blocks:
+        - type: card
+          config:
+            icon: /assets/icons/mesh.svg
+            title: Konnect (Beta)
+            description: |
+              Konnect regularly ships new features, and they start out as a _beta_ release. In order to provide early access to these features using Terraform, we provide the `konnect-beta` provider.
+
+              The **beta** provider currently supports:
```

```suggestion
              The **BETA** provider currently supports:
```
Consistency with the following cards